src/linked_list.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+
+
+class Node(object):
+    def __init__(self, value):
+        """Create an instance of Node."""
+        self.value = value
+        self.next_node = None
+
+
+class LinkedList(object):
+    def __init__(self, param=None):
+        """Create an instance of LinkedList."""
+        self.head_node = None
+        self.length_list = 0
+        if param:
+            for i in param:
+                self.push(i)
+
+    def push(self, val):
+        """Insert a new node to the head of a linked list."""
+        new_node = Node(val)
+        new_node.next_node = self.head_node
+        self.head_node = new_node
+        self.length_list += 1
+        return self.head_node
```

Comment:
We also used the return value in the unit test (making sure that we in fact changed the head by .push())